system.json

```
@@ -51,36 +51,67 @@
     },
     "Item": {
       "weapon": {
+        "filePathFields": {
+          "activities.*.img": ["IMAGE"]
+        },
         "htmlFields": ["description.value", "description.chat", "unidentified.description"]
       },
       "equipment": {
+        "filePathFields": {
+          "activities.*.img": ["IMAGE"]
+        },
         "htmlFields": ["description.value", "description.chat", "unidentified.description"]
       },
       "consumable": {
+        "filePathFields": {
+          "activities.*.img": ["IMAGE"]
+        },
         "htmlFields": ["description.value", "description.chat", "unidentified.description"]
       },
       "tool": {
+        "filePathFields": {
+          "activities.*.img": ["IMAGE"]
+        },
         "htmlFields": ["description.value", "description.chat", "unidentified.description"]
       },
       "loot": {
         "htmlFields": ["description.value", "description.chat", "unidentified.description"]
       },
       "race": {
+        "filePathFields": {
+          "advancement.*.img": ["IMAGE"]
```

Advancement uses `icon`, not `img`:

```suggestion
          "advancement.*.icon": ["IMAGE"]
```